### ISSUE TITLE
How to make a simple sniffer?

### ISSUE THREAD

**Issue by alessio31183** [jgeisler0303/CANFestivino#6]
First of all that for the great job !
What I have to do if I just wish to make a sniffer able to read data on the BUS with CANopen protocol taking values from external sensor (Inclinometer)?

**Comment by jgeisler0303** [comment]
Well, CANopen is a pretty complicated protocol. If you really want to make a "sniffer" I would suggest you pass the raw CAN data to a PC and do the protocol stuff there.

Alternatively, if you just want to read one sensor, you need to know how access that sensors data (PDO, SDO, cobID). Then you can configure CANFestivino to read (pull, listen) that data and pass it on. If you don't know what PDO, SDO or cobID is, I suggest you first read up on how CANopen works.

**Comment by alessio31183** [comment]
yes I'm trying to figure out better how the project works (dictionary making, functions....)
